src/databricks/labs/ucx/source_code/python_libraries.py

```diff
@@ -71,12 +75,25 @@ def _install_library(self, path_lookup: PathLookup, library: Path) -> MaybeDepen
         # invoke pip install via subprocess to install this library into self._lib_install_folder
         venv = self._temporary_virtual_environment(path_lookup).as_posix()
         existing_packages = os.listdir(venv)
-        pip_install_arguments = ["pip", "install", library.name, "-t", venv]
-        try:
-            subprocess.run(pip_install_arguments, check=True)
-        except CalledProcessError as e:
-            problem = DependencyProblem("library-install-failed", f"Failed to install {library}: {e}")
-            return MaybeDependency(None, [problem])
+        if library.suffix == ".egg":
+            # Setup requires the install directory to be in the python path
+            python_path = os.environ.get("PYTHONPATH", "")
```

Comment:
A requirement from `setup`/`easy_install` to install in a different location using the `-d` flag. It can only be installed in directories on the python path